adage/genes/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+from .models import (
+    Gene
+)
```

Comment:
Same comment here as in `analyze/serializers.py`. I am confused why this isn't a one-line import.

Reply:
I'll change it.